addons/artillerytables/stringtable.xml

```diff
@@ -45,9 +45,9 @@
             <Turkish>Hava Direnci</Turkish>
         </Key>
         <Key ID="STR_ACE_ArtilleryTables_advancedCorrections_description">
-            <English>For Player Shots, Model Air Resistance and Wind Effects</English>
+            <English>For player shots, model air resistance and wind effects</English>
```

Comment:
No need to put everything in small case. Besides your PR references only a Spanish translation

Reply:
No, I renamed the PR inappropriately, fixed.